spec/acceptance/suites/default/10_rsyslog_integration_spec.rb

```diff
@@ -41,507 +43,18 @@
 
 syslog_servers = hosts_with_role(hosts, 'syslog_server')
 non_syslog_servers = hosts - syslog_servers
-
+master      = only_host_with_role(hosts, 'master')
```

Comment:
FYI `master` is a method that returns the host with role 'master'.  See https://www.rubydoc.info/gems/beaker/Beaker/DSL/Roles#master-instance_method